Mon Jul 29 14:21:13 CST 2024 /Users/jrc/develop/timelink-kleio/tests
Comparing translation results.
Mon Jul 29 14:28:05 CST 2024 /Users/jrc/develop/timelink-kleio/tests
Only in kleio-home/sources/reference_translations: .gitignore
diff -r kleio-home/sources/reference_translations/bugs/bugs.err kleio-home/sources/test_translations/bugs/bugs.err
diff -r kleio-home/sources/reference_translations/bugs/bugs.files.json kleio-home/sources/test_translations/bugs/bugs.files.json
diff -r kleio-home/sources/reference_translations/bugs/bugs.rpt kleio-home/sources/test_translations/bugs/bugs.rpt
diff -r kleio-home/sources/reference_translations/bugs/bugs.xml kleio-home/sources/test_translations/bugs/bugs.xml
diff -r kleio-home/sources/reference_translations/identifications/mhk_identification_toliveira.err kleio-home/sources/test_translations/identifications/mhk_identification_toliveira.err
diff -r kleio-home/sources/reference_translations/identifications/mhk_identification_toliveira.files.json kleio-home/sources/test_translations/identifications/mhk_identification_toliveira.files.json
diff -r kleio-home/sources/reference_translations/identifications/mhk_identification_toliveira.rpt kleio-home/sources/test_translations/identifications/mhk_identification_toliveira.rpt
diff -r kleio-home/sources/reference_translations/identifications/mhk_identification_toliveira.xml kleio-home/sources/test_translations/identifications/mhk_identification_toliveira.xml
diff -r kleio-home/sources/reference_translations/issues/issue21.files.json kleio-home/sources/test_translations/issues/issue21.files.json
diff -r kleio-home/sources/reference_translations/issues/issue21.rpt kleio-home/sources/test_translations/issues/issue21.rpt
diff -r kleio-home/sources/reference_translations/issues/issue21.xml kleio-home/sources/test_translations/issues/issue21.xml
diff -r kleio-home/sources/reference_translations/issues/issue34.cli kleio-home/sources/test_translations/issues/issue34.cli
1,2c1
< kleio$gacto2.str
<     property$multiple-entry-flag/124
<     fonte$issue34/2024
>    property$multiple-entry-flag/124/id=root-pro1
>    fonte$issue34/2024
<         lista$mutiple-entries/0/0/0
>       lista$mutiple-entries/0/0/0
<          n$joaquim
<             ls$nomes/joaquim|manuel|costa|ramos|carvalho/obs=neste caso
<             usamos o caracter ascii 124 para separar diferentes valores para o mesmo elemento
<             ls$nomes/joaquim;manuel;costa;ramos;carvalho/obs=neste caso
<             o ponto e virgula não separa os nomes
>          n$joaquim/id=mutiple-entries-per1
>             ls$nomes/joaquim|manuel|costa|ramos|carvalho/obs=neste caso usamos o caracter ascii 124 para separar diferentes valores para o mesmo elemento
>             ls$nomes/joaquim;manuel;costa;ramos;carvalho/obs=neste caso o ponto e virgula não separa os nomes
<         b$b1788.1118/22/1/1788/vigario luis barreto de figueiredo castilho e o reverendo cosme dias ribeiro de coimbra
>       b$b1788.1118/22/1/1788/vigario luis barreto de figueiredo castilho e o reverendo cosme dias ribeiro de coimbra
<             n$bernardo/m/obs=diz a margem casais; fl.10v/id=b1788.1118-per1
<                ls$datanasc/17880111
\ No newline at end of file
>          n$bernardo/m/obs=diz a margem casais; fl.10v/id=b1788.1118-per1
>             ls$datanasc/17880111
diff -r kleio-home/sources/reference_translations/issues/issue34.err kleio-home/sources/test_translations/issues/issue34.err
< 1  errors. 
> 0  errors. 
diff -r kleio-home/sources/reference_translations/issues/issue34.files.json kleio-home/sources/test_translations/issues/issue34.files.json
<   "errors":1,
>   "errors":0,
Only in kleio-home/sources/reference_translations/issues: issue34.ids
Only in kleio-home/sources/test_translations/issues: issue34.org
diff -r kleio-home/sources/reference_translations/issues/issue34.rpt kleio-home/sources/test_translations/issues/issue34.rpt
18,22d17
< 
< ERROR: issue34.cli line 3 ** INTERNAL ERROR: Failure: db_store
< Near lines: 3 
< Near lines: 4     fonte$issue34/2024
< 
< 1  errors. 
> 0  errors. 
diff -r kleio-home/sources/reference_translations/issues/issue34.xml kleio-home/sources/test_translations/issues/issue34.xml
<          <core><![CDATA[joaquim|manuel|costa|ramos|carvalho]]></core>   </ELEMENT>
>          <core><![CDATA[joaquim]]></core>      <core><![CDATA[manuel]]></core>      <core><![CDATA[costa]]></core>      <core><![CDATA[ramos]]></core>      <core><![CDATA[carvalho]]></core>   </ELEMENT>
<          <core><![CDATA[joaquim]]></core>      <core><![CDATA[manuel]]></core>      <core><![CDATA[costa]]></core>      <core><![CDATA[ramos]]></core>      <core><![CDATA[carvalho]]></core>   </ELEMENT>
>          <core><![CDATA[joaquim;manuel;costa;ramos;carvalho]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl.10v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl.10v]]></core>   </ELEMENT>
diff -r kleio-home/sources/reference_translations/issues/issue34b.files.json kleio-home/sources/test_translations/issues/issue34b.files.json
diff -r kleio-home/sources/reference_translations/issues/issue34b.rpt kleio-home/sources/test_translations/issues/issue34b.rpt
diff -r kleio-home/sources/reference_translations/issues/issue34b.xml kleio-home/sources/test_translations/issues/issue34b.xml
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-a.err kleio-home/sources/test_translations/linked_data/dehergne-a.err
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-a.files.json kleio-home/sources/test_translations/linked_data/dehergne-a.files.json
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-a.rpt kleio-home/sources/test_translations/linked_data/dehergne-a.rpt
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-a.xml kleio-home/sources/test_translations/linked_data/dehergne-a.xml
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-locations-1644.err kleio-home/sources/test_translations/linked_data/dehergne-locations-1644.err
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-locations-1644.files.json kleio-home/sources/test_translations/linked_data/dehergne-locations-1644.files.json
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-locations-1644.rpt kleio-home/sources/test_translations/linked_data/dehergne-locations-1644.rpt
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-locations-1644.xml kleio-home/sources/test_translations/linked_data/dehergne-locations-1644.xml
diff -r kleio-home/sources/reference_translations/linked_data/linked-datanw.err kleio-home/sources/test_translations/linked_data/linked-datanw.err
diff -r kleio-home/sources/reference_translations/linked_data/linked-datanw.files.json kleio-home/sources/test_translations/linked_data/linked-datanw.files.json
diff -r kleio-home/sources/reference_translations/linked_data/linked-datanw.rpt kleio-home/sources/test_translations/linked_data/linked-datanw.rpt
diff -r kleio-home/sources/reference_translations/linked_data/linked-datanw.xml kleio-home/sources/test_translations/linked_data/linked-datanw.xml
diff -r kleio-home/sources/reference_translations/linked_data/multiplelinks.err kleio-home/sources/test_translations/linked_data/multiplelinks.err
diff -r kleio-home/sources/reference_translations/linked_data/multiplelinks.files.json kleio-home/sources/test_translations/linked_data/multiplelinks.files.json
diff -r kleio-home/sources/reference_translations/linked_data/multiplelinks.rpt kleio-home/sources/test_translations/linked_data/multiplelinks.rpt
diff -r kleio-home/sources/reference_translations/linked_data/multiplelinks.xml kleio-home/sources/test_translations/linked_data/multiplelinks.xml
diff -r kleio-home/sources/reference_translations/notariais/docsregiospontepisc.err kleio-home/sources/test_translations/notariais/docsregiospontepisc.err
diff -r kleio-home/sources/reference_translations/notariais/docsregiospontepisc.files.json kleio-home/sources/test_translations/notariais/docsregiospontepisc.files.json
diff -r kleio-home/sources/reference_translations/notariais/docsregiospontepisc.rpt kleio-home/sources/test_translations/notariais/docsregiospontepisc.rpt
diff -r kleio-home/sources/reference_translations/notariais/docsregiospontepisc.xml kleio-home/sources/test_translations/notariais/docsregiospontepisc.xml
diff -r kleio-home/sources/reference_translations/notariais/notarial80.err kleio-home/sources/test_translations/notariais/notarial80.err
diff -r kleio-home/sources/reference_translations/notariais/notarial80.files.json kleio-home/sources/test_translations/notariais/notarial80.files.json
diff -r kleio-home/sources/reference_translations/notariais/notarial80.rpt kleio-home/sources/test_translations/notariais/notarial80.rpt
diff -r kleio-home/sources/reference_translations/notariais/notarial80.xml kleio-home/sources/test_translations/notariais/notarial80.xml
diff -r kleio-home/sources/reference_translations/notariais/notlousa1.err kleio-home/sources/test_translations/notariais/notlousa1.err
diff -r kleio-home/sources/reference_translations/notariais/notlousa1.files.json kleio-home/sources/test_translations/notariais/notlousa1.files.json
diff -r kleio-home/sources/reference_translations/notariais/notlousa1.rpt kleio-home/sources/test_translations/notariais/notlousa1.rpt
diff -r kleio-home/sources/reference_translations/notariais/notlousa1.xml kleio-home/sources/test_translations/notariais/notlousa1.xml
diff -r kleio-home/sources/reference_translations/notariais/perdoes.err kleio-home/sources/test_translations/notariais/perdoes.err
diff -r kleio-home/sources/reference_translations/notariais/perdoes.files.json kleio-home/sources/test_translations/notariais/perdoes.files.json
diff -r kleio-home/sources/reference_translations/notariais/perdoes.rpt kleio-home/sources/test_translations/notariais/perdoes.rpt
diff -r kleio-home/sources/reference_translations/notariais/perdoes.xml kleio-home/sources/test_translations/notariais/perdoes.xml
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bap-com-celebrantes.files.json kleio-home/sources/test_translations/paroquiais/baptismos/bap-com-celebrantes.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bap-com-celebrantes.rpt kleio-home/sources/test_translations/paroquiais/baptismos/bap-com-celebrantes.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bap-com-celebrantes.xml kleio-home/sources/test_translations/paroquiais/baptismos/bap-com-celebrantes.xml
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapt1714.err kleio-home/sources/test_translations/paroquiais/baptismos/bapt1714.err
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapt1714.files.json kleio-home/sources/test_translations/paroquiais/baptismos/bapt1714.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapt1714.rpt kleio-home/sources/test_translations/paroquiais/baptismos/bapt1714.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapt1714.xml kleio-home/sources/test_translations/paroquiais/baptismos/bapt1714.xml
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapteirasproblem1.err kleio-home/sources/test_translations/paroquiais/baptismos/bapteirasproblem1.err
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapteirasproblem1.files.json kleio-home/sources/test_translations/paroquiais/baptismos/bapteirasproblem1.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapteirasproblem1.rpt kleio-home/sources/test_translations/paroquiais/baptismos/bapteirasproblem1.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapteirasproblem1.xml kleio-home/sources/test_translations/paroquiais/baptismos/bapteirasproblem1.xml
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 9v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 9v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 10]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 10]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 10]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 10]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 10-10v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 10-10v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 10v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 10v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl.10v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl.10v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margemescravote]]></core>      <core><![CDATA[ fl. 10v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margemescravote; fl. 10v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 11]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 11]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 11]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 11]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 11]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 11]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 11v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 11v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 11v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 11v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 11v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 11v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 11v-12]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 11v-12]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 12]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 12]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 12]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 12]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem carvalho]]></core>      <core><![CDATA[ fl. 12]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem carvalho; fl. 12]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 12-12v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 12-12v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 12v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 12v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 12v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 12v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 12v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 12v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 13]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 13]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 13]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 13]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 13]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 13]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 13]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 13]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem murtal]]></core>      <core><![CDATA[ fl. 13v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem murtal; fl. 13v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 13v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 13v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 13v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 13v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl.13v-14]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl.13v-14]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 14]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 14]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 14]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 14]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 14]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 14]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 14-14v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 14-14v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 14v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 14v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margemeiras]]></core>      <core><![CDATA[ fl. 14v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margemeiras; fl. 14v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl.14v-15]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl.14v-15]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 15]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 15]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 15]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 15]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 15]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 15]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem azenha]]></core>      <core><![CDATA[ fl.15-15v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem azenha; fl.15-15v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 15v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 15v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 15v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 15v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 15v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 15v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 16]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 16]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 16]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 16]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 16]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 16]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 16]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 16]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 16v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 16v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 16v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 16v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 16v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 16v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 16v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 16v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 17]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 17]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 17]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 17]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 17]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 17]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 17-17v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 17-17v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem rocha velha]]></core>      <core><![CDATA[ fl. 17v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem rocha velha; fl. 17v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 17v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 17v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 17v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 17v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 17v-18]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 17v-18]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem lordemao]]></core>      <core><![CDATA[ fl. 18]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem lordemao; fl. 18]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 18]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 18]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 18]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 18]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 18]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 18]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 18v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 18v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 18v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 18v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 18v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 18v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 18v-19]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 18v-19]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 19]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 19]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 19]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 19]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 19]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 19]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 19v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 19v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem murtal]]></core>      <core><![CDATA[ fl. 19v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem murtal; fl. 19v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem carvalho]]></core>      <core><![CDATA[ fl. 19v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem carvalho; fl. 19v]]></core>   </ELEMENT>
<          <core><![CDATA[antonio jose fernandes da silva, encomendado e o padre jeronimo fernandes prata]]></core>      <core><![CDATA[ o vigario morreu]]></core>   </ELEMENT>
>          <core><![CDATA[antonio jose fernandes da silva, encomendado e o padre jeronimo fernandes prata; o vigario morreu]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem murtal]]></core>      <core><![CDATA[ fl. 20]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem murtal; fl. 20]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 20]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 20]]></core>   </ELEMENT>
<          <core><![CDATA[vigario luis barreto de figueiredo castilho e o padre jeronimo fernandes prata]]></core>      <core><![CDATA[ feito por antonio jose fernandes da silva]]></core>   </ELEMENT>
>          <core><![CDATA[vigario luis barreto de figueiredo castilho e o padre jeronimo fernandes prata; feito por antonio jose fernandes da silva]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 20]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 20]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem vilarinho]]></core>      <core><![CDATA[ fl. 20v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem vilarinho; fl. 20v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem vilarinho]]></core>      <core><![CDATA[ fl. 20v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem vilarinho; fl. 20v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 20v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 20v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 20v-21]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 20v-21]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 20]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 20]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 20-21v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 20-21v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 21v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 21v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl.21v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl.21v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 21]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 21]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 21]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 21]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 22-22v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 22-22v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 22-22v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 22-22v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 22v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 22v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 22v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 22v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 22v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 22v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 23]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 23]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 23]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 23]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 23-23v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 23-23v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 23v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 23v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 23v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 23v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ fl. 23v-24]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; fl. 23v-24]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 24]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 24]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 24]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 24]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 24v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 24v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 24v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 24v]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 25]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 25]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 25]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 25]]></core>   </ELEMENT>
<          <core><![CDATA[diz a margem eiras]]></core>      <core><![CDATA[ fl. 25v]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem eiras; fl. 25v]]></core>   </ELEMENT>
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapteirasproblem2.err kleio-home/sources/test_translations/paroquiais/baptismos/bapteirasproblem2.err
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapteirasproblem2.files.json kleio-home/sources/test_translations/paroquiais/baptismos/bapteirasproblem2.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapteirasproblem2.rpt kleio-home/sources/test_translations/paroquiais/baptismos/bapteirasproblem2.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapteirasproblem2.xml kleio-home/sources/test_translations/paroquiais/baptismos/bapteirasproblem2.xml
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/rogeriodomingos_bap_problem.err kleio-home/sources/test_translations/paroquiais/baptismos/rogeriodomingos_bap_problem.err
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/rogeriodomingos_bap_problem.files.json kleio-home/sources/test_translations/paroquiais/baptismos/rogeriodomingos_bap_problem.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/rogeriodomingos_bap_problem.rpt kleio-home/sources/test_translations/paroquiais/baptismos/rogeriodomingos_bap_problem.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/rogeriodomingos_bap_problem.xml kleio-home/sources/test_translations/paroquiais/baptismos/rogeriodomingos_bap_problem.xml
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.err kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.err
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.files.json kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.rpt kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.xml kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.xml
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722.err kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722.err
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722.files.json kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722.rpt kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722.xml kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722.xml
diff -r kleio-home/sources/reference_translations/paroquiais/crisma/crisma1753.err kleio-home/sources/test_translations/paroquiais/crisma/crisma1753.err
diff -r kleio-home/sources/reference_translations/paroquiais/crisma/crisma1753.files.json kleio-home/sources/test_translations/paroquiais/crisma/crisma1753.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/crisma/crisma1753.rpt kleio-home/sources/test_translations/paroquiais/crisma/crisma1753.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/crisma/crisma1753.xml kleio-home/sources/test_translations/paroquiais/crisma/crisma1753.xml
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688.err kleio-home/sources/test_translations/paroquiais/obitos/ob1688.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688.files.json kleio-home/sources/test_translations/paroquiais/obitos/ob1688.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688.rpt kleio-home/sources/test_translations/paroquiais/obitos/ob1688.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688.xml kleio-home/sources/test_translations/paroquiais/obitos/ob1688.xml
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688b.err kleio-home/sources/test_translations/paroquiais/obitos/ob1688b.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688b.files.json kleio-home/sources/test_translations/paroquiais/obitos/ob1688b.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688b.rpt kleio-home/sources/test_translations/paroquiais/obitos/ob1688b.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688b.xml kleio-home/sources/test_translations/paroquiais/obitos/ob1688b.xml
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obit1714.err kleio-home/sources/test_translations/paroquiais/obitos/obit1714.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obit1714.files.json kleio-home/sources/test_translations/paroquiais/obitos/obit1714.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obit1714.rpt kleio-home/sources/test_translations/paroquiais/obitos/obit1714.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obit1714.xml kleio-home/sources/test_translations/paroquiais/obitos/obit1714.xml
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obiteirasproblem.err kleio-home/sources/test_translations/paroquiais/obitos/obiteirasproblem.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obiteirasproblem.files.json kleio-home/sources/test_translations/paroquiais/obitos/obiteirasproblem.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obiteirasproblem.rpt kleio-home/sources/test_translations/paroquiais/obitos/obiteirasproblem.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obiteirasproblem.xml kleio-home/sources/test_translations/paroquiais/obitos/obiteirasproblem.xml
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort.err kleio-home/sources/test_translations/paroquiais/obitos/obitoShort.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort.files.json kleio-home/sources/test_translations/paroquiais/obitos/obitoShort.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort.rpt kleio-home/sources/test_translations/paroquiais/obitos/obitoShort.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort.xml kleio-home/sources/test_translations/paroquiais/obitos/obitoShort.xml
<          <core><![CDATA[torre de vilela]]></core>      <core><![CDATA[e mais nao disse]]></core>      <core><![CDATA[e al nao disse]]></core>         <original><![CDATA[bilela]]></original>
>          <core><![CDATA[torre de vilela;e mais nao disse;e al nao disse;]]></core>         <original><![CDATA[bilela;]]></original>
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id.err kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id.files.json kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id.rpt kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id.xml kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id.xml
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id_id.err kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id_id.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id_id.files.json kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id_id.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id_id.rpt kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id_id.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id_id.xml kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id_id.xml
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/testeParentescos.err kleio-home/sources/test_translations/paroquiais/obitos/testeParentescos.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/testeParentescos.files.json kleio-home/sources/test_translations/paroquiais/obitos/testeParentescos.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/testeParentescos.rpt kleio-home/sources/test_translations/paroquiais/obitos/testeParentescos.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/testeParentescos.xml kleio-home/sources/test_translations/paroquiais/obitos/testeParentescos.xml
diff -r kleio-home/sources/reference_translations/roisdeconfessados/coja-rol-1841.err kleio-home/sources/test_translations/roisdeconfessados/coja-rol-1841.err
diff -r kleio-home/sources/reference_translations/roisdeconfessados/coja-rol-1841.files.json kleio-home/sources/test_translations/roisdeconfessados/coja-rol-1841.files.json
diff -r kleio-home/sources/reference_translations/roisdeconfessados/coja-rol-1841.rpt kleio-home/sources/test_translations/roisdeconfessados/coja-rol-1841.rpt
diff -r kleio-home/sources/reference_translations/roisdeconfessados/coja-rol-1841.xml kleio-home/sources/test_translations/roisdeconfessados/coja-rol-1841.xml
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol1.err kleio-home/sources/test_translations/roisdeconfessados/rol1.err
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol1.files.json kleio-home/sources/test_translations/roisdeconfessados/rol1.files.json
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol1.rpt kleio-home/sources/test_translations/roisdeconfessados/rol1.rpt
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol1.xml kleio-home/sources/test_translations/roisdeconfessados/rol1.xml
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol2.err kleio-home/sources/test_translations/roisdeconfessados/rol2.err
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol2.files.json kleio-home/sources/test_translations/roisdeconfessados/rol2.files.json
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol2.rpt kleio-home/sources/test_translations/roisdeconfessados/rol2.rpt
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol2.xml kleio-home/sources/test_translations/roisdeconfessados/rol2.xml
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol3.err kleio-home/sources/test_translations/roisdeconfessados/rol3.err
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol3.files.json kleio-home/sources/test_translations/roisdeconfessados/rol3.files.json
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol3.rpt kleio-home/sources/test_translations/roisdeconfessados/rol3.rpt
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol3.xml kleio-home/sources/test_translations/roisdeconfessados/rol3.xml
diff -r kleio-home/sources/reference_translations/varia/Devedores.err kleio-home/sources/test_translations/varia/Devedores.err
diff -r kleio-home/sources/reference_translations/varia/Devedores.files.json kleio-home/sources/test_translations/varia/Devedores.files.json
diff -r kleio-home/sources/reference_translations/varia/Devedores.rpt kleio-home/sources/test_translations/varia/Devedores.rpt
diff -r kleio-home/sources/reference_translations/varia/Devedores.xml kleio-home/sources/test_translations/varia/Devedores.xml
diff -r kleio-home/sources/reference_translations/varia/EmpFAfonso.err kleio-home/sources/test_translations/varia/EmpFAfonso.err
diff -r kleio-home/sources/reference_translations/varia/EmpFAfonso.files.json kleio-home/sources/test_translations/varia/EmpFAfonso.files.json
diff -r kleio-home/sources/reference_translations/varia/EmpFAfonso.rpt kleio-home/sources/test_translations/varia/EmpFAfonso.rpt
diff -r kleio-home/sources/reference_translations/varia/EmpFAfonso.xml kleio-home/sources/test_translations/varia/EmpFAfonso.xml
diff -r kleio-home/sources/reference_translations/varia/auc-alunos-264605-A-140337-140771.err kleio-home/sources/test_translations/varia/auc-alunos-264605-A-140337-140771.err
diff -r kleio-home/sources/reference_translations/varia/auc-alunos-264605-A-140337-140771.files.json kleio-home/sources/test_translations/varia/auc-alunos-264605-A-140337-140771.files.json
diff -r kleio-home/sources/reference_translations/varia/auc-alunos-264605-A-140337-140771.rpt kleio-home/sources/test_translations/varia/auc-alunos-264605-A-140337-140771.rpt
diff -r kleio-home/sources/reference_translations/varia/auc-alunos-264605-A-140337-140771.xml kleio-home/sources/test_translations/varia/auc-alunos-264605-A-140337-140771.xml
diff -r kleio-home/sources/reference_translations/varia/auc_cartulario18.err kleio-home/sources/test_translations/varia/auc_cartulario18.err
diff -r kleio-home/sources/reference_translations/varia/auc_cartulario18.files.json kleio-home/sources/test_translations/varia/auc_cartulario18.files.json
diff -r kleio-home/sources/reference_translations/varia/auc_cartulario18.rpt kleio-home/sources/test_translations/varia/auc_cartulario18.rpt
diff -r kleio-home/sources/reference_translations/varia/auc_cartulario18.xml kleio-home/sources/test_translations/varia/auc_cartulario18.xml
diff -r kleio-home/sources/reference_translations/varia/cartas.err kleio-home/sources/test_translations/varia/cartas.err
diff -r kleio-home/sources/reference_translations/varia/cartas.files.json kleio-home/sources/test_translations/varia/cartas.files.json
diff -r kleio-home/sources/reference_translations/varia/cartas.rpt kleio-home/sources/test_translations/varia/cartas.rpt
diff -r kleio-home/sources/reference_translations/varia/cartas.xml kleio-home/sources/test_translations/varia/cartas.xml
diff -r kleio-home/sources/reference_translations/varia/eleicoes1.err kleio-home/sources/test_translations/varia/eleicoes1.err
diff -r kleio-home/sources/reference_translations/varia/eleicoes1.files.json kleio-home/sources/test_translations/varia/eleicoes1.files.json
diff -r kleio-home/sources/reference_translations/varia/eleicoes1.rpt kleio-home/sources/test_translations/varia/eleicoes1.rpt
diff -r kleio-home/sources/reference_translations/varia/eleicoes1.xml kleio-home/sources/test_translations/varia/eleicoes1.xml
diff -r kleio-home/sources/reference_translations/varia/genealdiversas.err kleio-home/sources/test_translations/varia/genealdiversas.err
diff -r kleio-home/sources/reference_translations/varia/genealdiversas.files.json kleio-home/sources/test_translations/varia/genealdiversas.files.json
diff -r kleio-home/sources/reference_translations/varia/genealdiversas.rpt kleio-home/sources/test_translations/varia/genealdiversas.rpt
diff -r kleio-home/sources/reference_translations/varia/genealdiversas.xml kleio-home/sources/test_translations/varia/genealdiversas.xml
diff -r kleio-home/sources/reference_translations/varia/glopes-lap1561.err kleio-home/sources/test_translations/varia/glopes-lap1561.err
diff -r kleio-home/sources/reference_translations/varia/glopes-lap1561.files.json kleio-home/sources/test_translations/varia/glopes-lap1561.files.json
diff -r kleio-home/sources/reference_translations/varia/glopes-lap1561.rpt kleio-home/sources/test_translations/varia/glopes-lap1561.rpt
diff -r kleio-home/sources/reference_translations/varia/glopes-lap1561.xml kleio-home/sources/test_translations/varia/glopes-lap1561.xml
diff -r kleio-home/sources/reference_translations/varia/ivcc.cli kleio-home/sources/test_translations/varia/ivcc.cli
<                ls$dead-tomb/Igreja de Góis/geral|view|70290/obs=Buried at the Igreja de Góis,see p.40 cap. XIII http:||www.patrimoniocultural.gov.pt|pt|patrimonio|patrimonio-imovel|pesquisa-do-patrimonio|classificado-ou-em-vias-de-classificacao
>                ls$dead-tomb/Igreja de Góis/geral|view|70290/obs=Buried at the Igreja de Góis,see p.40 cap. XIII http:|www.patrimoniocultural.gov.pt|pt|patrimonio|patrimonio-imovel|pesquisa-do-patrimonio|classificado-ou-em-vias-de-classificacao
<                referido$Pantaleão de Sá/id=ivc-pantaleao-de-sa/obs=See https:||www.geni.com|people|Pantaleão-de-Sá-de-Menezes-capitão-de-Sofala|6000000017017918072
>                referido$Pantaleão de Sá/id=ivc-pantaleao-de-sa/obs=See https:|www.geni.com|people|Pantaleão-de-Sá-de-Menezes-capitão-de-Sofala|6000000017017918072
<                referida$Luísa/id=ivc-doc4-per2-92-per1-96/obs=See https:||www.geni.com|people|Luisa-de-Vasconcelos|6000000017017436413
>                referida$Luísa/id=ivc-doc4-per2-92-per1-96/obs=See https:|www.geni.com|people|Luisa-de-Vasconcelos|6000000017017436413
diff -r kleio-home/sources/reference_translations/varia/ivcc.err kleio-home/sources/test_translations/varia/ivcc.err
diff -r kleio-home/sources/reference_translations/varia/ivcc.files.json kleio-home/sources/test_translations/varia/ivcc.files.json
diff -r kleio-home/sources/reference_translations/varia/ivcc.rpt kleio-home/sources/test_translations/varia/ivcc.rpt
diff -r kleio-home/sources/reference_translations/varia/ivcc.xml kleio-home/sources/test_translations/varia/ivcc.xml
<          <core><![CDATA[geral|view|70290]]></core>   </ELEMENT>
>          <core><![CDATA[geral]]></core>      <core><![CDATA[view]]></core>      <core><![CDATA[70290]]></core>   </ELEMENT>
<          <core><![CDATA[Buried at the Igreja de Góis,see p.40 cap. XIII http:||www.patrimoniocultural.gov.pt|pt|patrimonio|patrimonio-imovel|pesquisa-do-patrimonio|classificado-ou-em-vias-de-classificacao]]></core>   </ELEMENT>
>          <core><![CDATA[Buried at the Igreja de Góis,see p.40 cap. XIII http:]]></core>      <core><![CDATA[www.patrimoniocultural.gov.pt]]></core>      <core><![CDATA[pt]]></core>      <core><![CDATA[patrimonio]]></core>      <core><![CDATA[patrimonio-imovel]]></core>      <core><![CDATA[pesquisa-do-patrimonio]]></core>      <core><![CDATA[classificado-ou-em-vias-de-classificacao]]></core>   </ELEMENT>
<          <core><![CDATA[See https:||www.geni.com|people|Pantaleão-de-Sá-de-Menezes-capitão-de-Sofala|6000000017017918072]]></core>   </ELEMENT>
>          <core><![CDATA[See https:]]></core>      <core><![CDATA[www.geni.com]]></core>      <core><![CDATA[people]]></core>      <core><![CDATA[Pantaleão-de-Sá-de-Menezes-capitão-de-Sofala]]></core>      <core><![CDATA[6000000017017918072]]></core>   </ELEMENT>
<          <core><![CDATA[See https:||www.geni.com|people|Luisa-de-Vasconcelos|6000000017017436413]]></core>   </ELEMENT>
>          <core><![CDATA[See https:]]></core>      <core><![CDATA[www.geni.com]]></core>      <core><![CDATA[people]]></core>      <core><![CDATA[Luisa-de-Vasconcelos]]></core>      <core><![CDATA[6000000017017436413]]></core>   </ELEMENT>
diff -r kleio-home/sources/reference_translations/varia/lrazao516pe.err kleio-home/sources/test_translations/varia/lrazao516pe.err
diff -r kleio-home/sources/reference_translations/varia/lrazao516pe.files.json kleio-home/sources/test_translations/varia/lrazao516pe.files.json
diff -r kleio-home/sources/reference_translations/varia/lrazao516pe.rpt kleio-home/sources/test_translations/varia/lrazao516pe.rpt
diff -r kleio-home/sources/reference_translations/varia/lrazao516pe.xml kleio-home/sources/test_translations/varia/lrazao516pe.xml
diff -r kleio-home/sources/reference_translations/varia/misMesa.err kleio-home/sources/test_translations/varia/misMesa.err
diff -r kleio-home/sources/reference_translations/varia/misMesa.files.json kleio-home/sources/test_translations/varia/misMesa.files.json
diff -r kleio-home/sources/reference_translations/varia/misMesa.rpt kleio-home/sources/test_translations/varia/misMesa.rpt
diff -r kleio-home/sources/reference_translations/varia/misMesa.xml kleio-home/sources/test_translations/varia/misMesa.xml
diff -r kleio-home/sources/reference_translations/varia/mp758-santiago-beduido.err kleio-home/sources/test_translations/varia/mp758-santiago-beduido.err
diff -r kleio-home/sources/reference_translations/varia/mp758-santiago-beduido.files.json kleio-home/sources/test_translations/varia/mp758-santiago-beduido.files.json
diff -r kleio-home/sources/reference_translations/varia/mp758-santiago-beduido.rpt kleio-home/sources/test_translations/varia/mp758-santiago-beduido.rpt
diff -r kleio-home/sources/reference_translations/varia/mp758-santiago-beduido.xml kleio-home/sources/test_translations/varia/mp758-santiago-beduido.xml
diff -r kleio-home/sources/reference_translations/varia/nommiz.err kleio-home/sources/test_translations/varia/nommiz.err
diff -r kleio-home/sources/reference_translations/varia/nommiz.files.json kleio-home/sources/test_translations/varia/nommiz.files.json
diff -r kleio-home/sources/reference_translations/varia/nommiz.rpt kleio-home/sources/test_translations/varia/nommiz.rpt
diff -r kleio-home/sources/reference_translations/varia/nommiz.xml kleio-home/sources/test_translations/varia/nommiz.xml
diff -r kleio-home/sources/reference_translations/varia/quotes.err kleio-home/sources/test_translations/varia/quotes.err
diff -r kleio-home/sources/reference_translations/varia/quotes.files.json kleio-home/sources/test_translations/varia/quotes.files.json
diff -r kleio-home/sources/reference_translations/varia/quotes.rpt kleio-home/sources/test_translations/varia/quotes.rpt
diff -r kleio-home/sources/reference_translations/varia/quotes.xml kleio-home/sources/test_translations/varia/quotes.xml
diff -r kleio-home/sources/reference_translations/varia/sourecoimbrabispcoimbra_apontamentosd.err kleio-home/sources/test_translations/varia/sourecoimbrabispcoimbra_apontamentosd.err
diff -r kleio-home/sources/reference_translations/varia/sourecoimbrabispcoimbra_apontamentosd.files.json kleio-home/sources/test_translations/varia/sourecoimbrabispcoimbra_apontamentosd.files.json
diff -r kleio-home/sources/reference_translations/varia/sourecoimbrabispcoimbra_apontamentosd.rpt kleio-home/sources/test_translations/varia/sourecoimbrabispcoimbra_apontamentosd.rpt
diff -r kleio-home/sources/reference_translations/varia/sourecoimbrabispcoimbra_apontamentosd.xml kleio-home/sources/test_translations/varia/sourecoimbrabispcoimbra_apontamentosd.xml
diff -r kleio-home/sources/reference_translations/varia/test-atr-date.err kleio-home/sources/test_translations/varia/test-atr-date.err
diff -r kleio-home/sources/reference_translations/varia/test-atr-date.files.json kleio-home/sources/test_translations/varia/test-atr-date.files.json
diff -r kleio-home/sources/reference_translations/varia/test-atr-date.rpt kleio-home/sources/test_translations/varia/test-atr-date.rpt
diff -r kleio-home/sources/reference_translations/varia/test-atr-date.xml kleio-home/sources/test_translations/varia/test-atr-date.xml
diff -r kleio-home/sources/reference_translations/varia/vereacao.err kleio-home/sources/test_translations/varia/vereacao.err
diff -r kleio-home/sources/reference_translations/varia/vereacao.files.json kleio-home/sources/test_translations/varia/vereacao.files.json
diff -r kleio-home/sources/reference_translations/varia/vereacao.rpt kleio-home/sources/test_translations/varia/vereacao.rpt
diff -r kleio-home/sources/reference_translations/varia/vereacao.xml kleio-home/sources/test_translations/varia/vereacao.xml
diff -r kleio-home/sources/reference_translations/varia/xpto_comuns_soure.err kleio-home/sources/test_translations/varia/xpto_comuns_soure.err
diff -r kleio-home/sources/reference_translations/varia/xpto_comuns_soure.files.json kleio-home/sources/test_translations/varia/xpto_comuns_soure.files.json
diff -r kleio-home/sources/reference_translations/varia/xpto_comuns_soure.rpt kleio-home/sources/test_translations/varia/xpto_comuns_soure.rpt
diff -r kleio-home/sources/reference_translations/varia/xpto_comuns_soure.xml kleio-home/sources/test_translations/varia/xpto_comuns_soure.xml
Comparing translation results finished.
